frontend/src/pages/Vulnerability/vulnerabilityStyle.ts

```diff
@@ -32,10 +31,7 @@ export const Root = styled('div')(({ theme }) => ({
     }
   },
 
-  [`& .${classes.cardRight}`]: {
-    // marginLeft: '32px',
-    // width: '30%'
-  },
+  [`& .${classes.cardRight}`]: {},
```

Comment:
Is this necessary to keep this, since we commented code is not being used and is now removed.

Reply:
Removed unused styling class.